api4/team_test.go

```diff
@@ -1419,7 +1419,7 @@ func TestAddTeamMember(t *testing.T) {
 		app.TOKEN_TYPE_TEAM_INVITATION,
 		model.MapToJson(map[string]string{"teamId": team.Id}),
 	)
-	<-th.App.Srv.Store.Token().Save(token)
+	th.App.Srv.Store.Token().Save(token)
```

Comment:
Should I add require.NotNil to the possible err returned on lines like these?

Reply:
Yes, for test would be nice to add them. it would be actually, `require.Nil(t, err)`.